lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_testers_action.rb

```diff
@@ -0,0 +1,78 @@
+require 'fastlane/action'
+require 'fastlane_core/ui/ui'
+
+require_relative '../helper/firebase_app_distribution_helper'
+require_relative '../helper/firebase_app_distribution_auth_client'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionAddTestersAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        if blank?(params[:emails]) && blank?(params[:file])
+          UI.user_error!("Must specify `emails` or `file`.")
+        end
+
+        emails = get_value_from_value_or_file(params[:emails], params[:file]).split(',')
+
+        if emails.count > 1000
+          UI.user_error!("A maximum of 1000 testers can be added at a time.")
+        end
+
+        fad_api_client.add_testers(params[:project], emails)
+
+        UI.success("✅ Tester(s) successfully added.")
+      end
+
+      def self.description
+        "Create testers"
+      end
+
+      def self.authors
+        ["Tunde Agboola"]
+      end
+
+      # supports markdown.
+      def self.details
+        "Create testers"
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :project,
+                                      env_name: "FIREBASEAPPDISTRO_PROJECT_NUMBER",
+                                      description: "Your Firebase project number. You can find the project number in the Firebase console, on the General Settings page",
+                                      type: Integer),
+          FastlaneCore::ConfigItem.new(key: :emails,
+                                      env_name: "FIREBASEAPPDISTRO_ADD_TESTERS_EMAILS",
+                                      description: "Comma separated list of tester emails to be created. A maximum of 1000 testers can be created at a time",
+                                      optional: true,
+                                      type: String),
+          FastlaneCore::ConfigItem.new(key: :file,
```

Comment:
How about `:emails_file` to match the convention from the main action?

Reply:
We originally suggested something like that i.e. `testers_file` that but the API council thought the less verbose `file` would be better

Reply:
We originally suggested something like this i.e. `testers_file` but the API council thought the less verbose `file` would be better.